mod-tool/src/main/scala/at/forsyte/apalache/tla/tooling/opt/PassExecutorCmd.scala

```diff
@@ -41,11 +40,9 @@ abstract class PassExecutorCmd(name: String, description: String)
    * NOTE: It is not invoked automatically, and you should invoke it explicitly in your `Cmd` class' [[run]] method.
    */
   def setCommonOptions(): Unit = {
-    executor.passOptions.set("general.debug", debug)
+    executor.passOptions
+      .set("general.debug", configuration.getOrElse(throw new Exception("TODO: Move into static options")).debug)
     executor.passOptions.set("smt.prof", smtprof)
     executor.passOptions.set("general.features", features)
-
-    // TODO: Remove pass option, and just rely on OutputManager config
-    executor.passOptions.set("io.outdir", OutputManager.outDir)
```

Comment:
This options is not used anywhere outside of the `OutputManager` now, so we don't need it! :)